R/hp_datatree.R

```diff
@@ -0,0 +1,141 @@
+dt_root <- function(x) {
+  names(x$Get("level", filterFun = data.tree::isRoot))
+}
+
+dt_root_df <- function(x) {
+  x <- dt_root(x)
+  data.frame(
+    from = x,
+    to = x,
+    stringsAsFactors = FALSE
+  )
+}
+
+dt_collect <- function(x) {
+  requireNamespaceQuietStop("data.tree")
```

Comment:
It's checked already at the top level in `hp_datatree`. Do we need to check again here?

Reply:
I thought I added this because RCMD check wasn't passing, but I just removed and there seem to be no issues. I'll remove.